reactivesocket-client/src/main/java/io/reactivesocket/client/LoadBalancer.java

```diff
@@ -426,37 +502,58 @@ public void onSubscribe(Subscription subscription) {
         }
 
         @Override
-        public void onNext(List<ReactiveSocketFactory<SocketAddress>> newFactories) {
-            List<ReactiveSocketFactory<SocketAddress>> removed = computeRemoved(newFactories);
+        public void onNext(Collection<ReactiveSocketFactory<T>> newFactories) {
             synchronized (LoadBalancer.this) {
```

Comment:
This also means that the `select()` will be blocking the eventloop for the duration of the refresh, correct?